docs/full/README.md

```diff
@@ -228,12 +228,46 @@ reason. Below is a brief summary of these commands
 
 `version`: provides basic version information about the plugin, likely the only common command of broad use to the CLI user.
 
-`info`: provides metadata about the plugin that the CLI will use when presenting information about plugins or when performing lifecycle operations on them.
+`info`: provides metadata about the plugin that the CLI will use when presenting information about plugins or when
+performing lifecycle operations on them.
 
-`post-install`: provide a means for a plugin to optionally implement some logic to be invoked right after a plugin is installed.
+`post-install`: provide a means for a plugin to optionally implement some logic to be invoked right after a plugin is
+installed.
 
-`generate-docs`: generate a tree of documentation markdown files for the commands the plugin provides, typically used by the CLI's generate-all-docs command to produce command documentation for all installed plugins
+`generate-docs`: generate a tree of documentation markdown files for the commands the plugin provides, typically used by
+the CLI's generate-all-docs command to produce command documentation for all installed plugins
 
 `lint`: validate the command name and arguments to flag any new terms unaccounted for in the CLI taxonomy document
 
-More information about these commands are available in the [plugin contract](../plugindev/contract.md) section of the plugin development guide.
+More information about these commands are available in the [plugin contract](../plugindev/contract.md) section of the
+plugin development guide.
+
+## Secure plugin installation
+
+CLI verifies the identity and integrity of the plugin while installing the plugin
+from the repository. You can find more details in
+the [secure plugin installation proposal document](../design/secure-plugin-installation-design.md)
+
+### User experience
+
+CLI verifies cosign signature of the plugin inventory image present in the
+repository. If the signature verification is success, it would download the
+plugin inventory image on the user's machine and caches to improve the latency
```

Comment:
caches what?

Reply:
updated the doc to convey we cache the plugin inventory image.